### DISCUSSION TITLE
⛔ 🚀 Feature Request: Move the Top Bar Publish button to the left

### DISCUSSION THREAD

**Discussion by kimfull** [elementor/elementor#23586] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Top Bar Editor interface suggestion: move Publish Butoon from right to left (Please refer to attached image)

### Describe the solution you'd like

Top Bar Editor interface suggestion: move Publish Butoon from right to left (Please refer to attached image)

### Additional context

![2023-08-28_192430](https://github.com/elementor/elementor/assets/3871491/daf51eab-a429-4264-ba8c-8c9a81dc0ca5)

**Comment by nicholaszein** [comment]
Hello @kimfull!

Thank you for submitting your idea! 🙏 

Unfortunately this time we won't be able to accept your suggestions. This is because we have other plans for this area of the Editor Top Bar that can't accommodate the Publish button. This will be part of a future phase of enhancements to the Editor Top Bar we are planning to release.

We appreciate your feedback.

Kind regards
